app/models/concerns/normalize_column.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module NormalizeColumn
+  extend ActiveSupport::Concern
+
+  included do
+    def self.normalize_column(column_name)
+      before_validation -> do
+        value = self[column_name]
```

Comment:
```suggestion
        value = self[column_name]&.squish
```

Reply:
I'd leave it as-is. Doesn't change much either way. Thanks!